aloha/robot_utils.py

```diff
@@ -307,3 +253,132 @@ def enable_gravity_compensation(bot: InterbotixManipulatorXS):
 def disable_gravity_compensation(bot: InterbotixManipulatorXS):
     gravity_compensation = InterbotixGravityCompensationInterface(bot.core)
     gravity_compensation.disable()
+
+
+def load_yaml_file(config_type: str = "robot", name: str = "aloha_static") -> dict:
+    """
+    Loads configuration from a YAML file based on the specified type and name.
+
+    :param config_type: Type of configuration to load, e.g., 'robot' or 'task'. Defaults to 'robot'.
+    :param name: Name of the robot or task configuration to load. Defaults to 'aloha_static' for robots.
+    :return: The loaded configuration as a dictionary.
+    :raises FileNotFoundError: Raised if the specified configuration file does not exist.
+    :raises RuntimeError: Raised if there is an error loading the YAML file.
+    """
+
+    # Base path of the config directory using absolute path
+    base_path = os.path.abspath(os.path.join("..", "config"))
```

Comment:
This seems like a fragile way to do this. Should we pass in the absolute filepath instead?

Reply:
Used pathlib for handling path. Now the python script can be executed from any location.